sqlite/migrate.go

```diff
@@ -183,11 +184,52 @@ func GetLastMigrationVersion(db *sql.DB) (version uint, migrationTableExists boo
 
 	var lastMigration uint64 = 0
 	if migrationTableExists {
-		row = db.QueryRow("SELECT version FROM status_go_schema_migrations")
+		row = db.QueryRow(fmt.Sprintf("SELECT version FROM %s", migrationTableName))
 		err = row.Scan(&lastMigration)
 		if err != nil && err != sql.ErrNoRows {
 			return 0, true, err
 		}
 	}
 	return uint(lastMigration), migrationTableExists, nil
 }
+
+func CreateMigrationTable(db *sql.DB, name string, version uint) error {
+	row := db.QueryRow("SELECT exists(SELECT name FROM sqlite_master WHERE type='table' AND name=?)", name)
+	exists := false
+	err := row.Scan(&exists)
+	if err != nil && err != sql.ErrNoRows {
+		return err
+	}
+	if exists {
+		return nil
+	}
+
+	// Create the table and insert the version
+	createTable := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (version uint64, dirty bool);`, name)
+	if _, err := db.Exec(createTable); err != nil {
+		return err
+	}
+	createIndex := fmt.Sprintf(`CREATE UNIQUE INDEX IF NOT EXISTS version_unique ON %s (version);`, name)
+	if _, err := db.Exec(createIndex); err != nil {
+		return err
+	}
+	insertVersion := fmt.Sprintf(`INSERT INTO %s (version, dirty) VALUES (?, ?)`, name) // nolint:gosec
```

Comment:
How about this to avoid `// nolint`?
```suggestion
	insertVersion := fmt.Sprintf(`INSERT INTO %s (version, dirty)`, name) + "VALUES (?, ?)"
```